pt-BR/mail/chrome/messenger/messenger.properties

```diff
@@ -85,7 +85,7 @@ LoadingMailMsgForPrint=(Carregando conteúdo para a impressão)
 LoadingMailMsgForPrintPreview=(Carregando conteúdo para visualizar impressão)
 saveAttachmentFailed=Não foi possível salvar o anexo. Por favor, verifique o nome do arquivo e tente de novo.
 saveMessageFailed=Não foi possível salvar a mensagem. Por favor, verifique o nome do arquivo e tente de novo.
-notDownloaded=(Not Downloaded)
+notDownloaded=(Não baixado)
 fileExists=O arquivo “%S” já existe. Deseja substituí-lo?
 
 downloadingNewsgroups=Recebendo newsgroups para leitura offline…
```

Comment:
CORRECTIONS NEEDED it should be downloadingNewsgroups=Recebendo grupos de notícias para leitura offline…